src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,131 @@
+package duke;
+
+import java.util.Scanner;
+
+
+public class Ui {
+
+    protected TaskList tasks;
```

Comment:
Perhaps Ui class does not need to aware of the TaskList class? 
Can consider Ui to display a message based on the provided arguments.